modules/core/keeper/msg_server_test.go

```diff
@@ -419,6 +445,14 @@ func (suite *KeeperTestSuite) TestHandleAcknowledgePacket() {
 				// replay should not error as it is treated as a no-op
 				_, err := keeper.Keeper.Acknowledgement(*suite.chainA.App.GetIBCKeeper(), suite.chainA.GetContext(), msg)
 				suite.Require().NoError(err)
+
+				if replay {
```

Comment:
small nit, to be more consistent maybe we could make this a `tc.replay`, we seem to be mixing things like `tc.expPass` and standalone bools like `replay`.

Reply:
done